spread.yaml

```diff
@@ -696,7 +696,7 @@ suites:
         restore: |
             "$TESTSLIB"/prepare-restore.sh --restore-suite
 
-    tests/nested/:
+    tests/nested/classic/:
         summary: Tests for nested images
         # Test cases are not yet ported to Fedora/openSUSE that is why
         # we keep them disabled. A later PR will enable most tests and
```

Comment:
I'm not sure I get the comment fully - it says "A lter PR will enable most tests and drop this blacklist"  - AFAICT there is no blacklist just a different backend. Is the comment maybe outdated?